oauth2_http/javatests/com/google/auth/oauth2/CertificateIdentityPoolSubjectTokenSupplierTest.java

```diff
@@ -42,11 +42,14 @@
 import java.net.URL;
 import java.nio.charset.StandardCharsets;
 import java.nio.file.Files;
+import java.nio.file.NoSuchFileException;
```

Comment:
Cannot comment on code not part of this PR, thus leaving a note here.
Can you extend L34 and import specific classes used in `org.junit.Assert`? go/java-style#s3.3.1-wildcard-imports